React InstantSearch/algolia-insights/src/App.js

```diff
@@ -20,6 +20,8 @@ const searchClient = algoliasearch(
   'aadef574be1f9252bb48d4ea09b5cfe5'
 );
 
+const HitWithInsights = connectHitInsights(window.aa)(Hit);
```

Comment:
I'd move this closer to (probably just after) `Hit`, right now this order confused me

Reply:
I agree @Haroenv, it's probably better to move the `Hit` inside its own file.

Reply:
done here: 41da299